adhocracy4/comments_async/static/comments_async/comment_form.jsx

```diff
@@ -132,52 +135,66 @@ export default class CommentForm extends React.Component {
-            id={'id_textarea-top' + this.props.commentId}
-            aria-describedby={this.state.showCommentError ? 'alert' : ('id_char-count' + this.props.commentId)}
-            aria-invalid={this.state.showCommentError}
-            className={this.props.commentCategoryChoices ? 'a4-comments__textarea--small form-group' : 'a4-comments__textarea--big form-group'}
-            placeholder={hasParent ? translated.yourReply : translated.yourComment}
-            onChange={this.handleTextChange.bind(this)}
-            value={this.state.comment}
-            onInput={this.handleTextareaGrow.bind(this)}
-            style={textareaStyle}
-          />
-          <div className="row">
-            <div className="col-12 col-sm-6">
+        <>
+          <h2 className="a4-comments_comment-form__heading-comments-allowed">{translated.formHeadingCommentsAllowed}</h2>
+          <form id={'id-comment-form' + this.props.commentId} className="a4-comments__comment-form__form" onSubmit={this.handleSubmit.bind(this)}>
+            {this.props.error &&
+              <Alert type="danger" message={this.props.errorMessage} onClick={this.props.handleErrorClick} />}
+            {this.props.commentCategoryChoices && !hasParent &&
+              <div>
+                <CategoryList
+                  idPrefix={this.props.commentId ? this.props.commentId : 'new'}
+                  categoriesChecked={this.state.selectedCategories}
+                  categoryChoices={this.props.commentCategoryChoices}
+                  handleControlFunc={this.handleCategorySelection.bind(this)}
+                />
+              </div>}
+            <div className="form-group" id={'group_id_textarea-top' + this.props.commentId}>
+              <label htmlFor={'id_textarea-top' + this.props.commentId}>
+                {translated.yourComment}
+              </label>
+              <textarea
+                id={'id_textarea-top' + this.props.commentId}
+                name={'id_textarea-top' + this.props.commentId}
+                aria-describedby={this.state.showCommentError ? 'alert' : ('id_char-count' + this.props.commentId)}
+                aria-invalid={this.state.showCommentError}
+                className={'form-control ' + this.props.commentCategoryChoices ? 'a4-comments__textarea--small' : 'a4-comments__textarea--big'}
+                onChange={this.handleTextChange.bind(this)}
+                value={this.state.comment}
+                onInput={this.handleTextareaGrow.bind(this)}
+                style={textareaStyle}
+              />
               <p
                 id={'id_char-count' + this.props.commentId}
                 className="a4-comments__char-count"
                 aria-live="polite"
               >
-                {this.state.commentCharCount}/4000{translated.characters}
+                <span aria-hidden="true">{this.state.commentCharCount}/4000<span className="a4-comments__char-count-word">{translated.characters}</span></span>
```

Comment:
@hom3mad3 not sure what is best here: making the word "Characters" optional (e.g. hiding it with display:none in mB as we don't want it there) means SRs will only read e.g. "5/4000" which is missing context. The two options I've tried are:
1. instead of display:none use a4-sr-only so it's only visible to SRs, however because of the span "Characters" is wrapped in it is no longer read as a complete sentence but separately which seems not ideal
2. using aria-hidden and adding a a separate span which doesn't contain the nested span. This seems to work better and it does read the whole sentence.

Feel free to change this if you have a better idea